### ISSUE TITLE
Better handling of media attachments

### ISSUE THREAD

**Issue by madduck** [wking/rss2email#69]
The RSS URL http://www.handelsblatt.com/contentexport/feed/wirtschaft has media files with every post, which result in mails such as:

```
![](http://www.handelsblatt.com/images/luftangriffe-auf-is-in-
syrien/12592958/2-format7.jpg?format=format7)Frankreich versucht, …
```

which is ugly and not really readable. Sure, I can post-process this away, but
I wonder why it's necessary to convert this in the first place?

**Comment by Ekleog** [comment]
Hello,

This repository has been deprecated for a few years now, and has been replaced by https://github.com/rss2email/rss2email .

If this issue is still relevant to you, and not fixed with v3.12.2, could you please reopen the issue there?

Cheers,
  Leo

**Comment by madduck** [comment]
This does not seem to be an issue anymore with the new version. Thanks.
